tests/test_utils.py

```diff
@@ -52,3 +52,19 @@ def test_get_all_field_names():
     assert isinstance(fields, list)
     # Convert to set as list could complain about ordering in equality check
     assert set(fields) == {'id', 'email', 'name'}
+
+
+def test_get_field_by_name():
+    field = utils.get_field_by_name(User, 'email')
```

Comment:
It looks good. I was thinking to avoid to look at `get_field_by_name` implementation we can destructure the tuple like this:

```python
field, model, direct, m2m = utils.get_field_by_name(User, 'email')
assert field == User._meta.get_field('email')
assert not model
assert direct
assert not m2m
```